modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchIndexSearcher.java

```diff
@@ -235,10 +235,8 @@ public long searchCount(SearchContext searchContext, Query query) {
 		}
 	}
 
-	@Override
-	@Reference(target = "(search.engine.impl=Elasticsearch)", unbind = "-")
 	public void setQuerySuggester(QuerySuggester querySuggester) {
-		super.setQuerySuggester(querySuggester);
+		_querySuggester = querySuggester;
```

Comment:
@W2jx This setQuerySuggester is only used in test, you should remove it and fix the test. You should never to have any code just for a test exposed.

Tina